README.md

```diff
@@ -41,17 +41,17 @@ _____________________________________
   * New Dad Left sing sprite 
   * Unused stage lights are now used
 ### Week 2:
-  * Both BF and Skid & Pump does "Hey!" animations
-  * Thunders does a quick light flash and zooms the camera in slightly
+  * Both BF Skid & Pump do a "Hey!" animation
```

Comment:
"Both BF and Skid & Pump" is more grammatically correct than this. Changing "does" to "do a" is correct, but the removal of "and" forms a list without commas. Previously, "BF" and "Skid & Pump" were treated as separate items in a list, now that distinction is unclear.

Reply:
> "Both BF and Skid & Pump" is more grammatically correct than this. Changing "does" to "do a" is correct, but the removal of "and" forms what appears to be a list without commas. Previously, "BF" and "Skid & Pump" were treated as separate items, now that distinction is unclear.

resolved